dev-vcs/git-cola/Manifest

```diff
@@ -2,3 +2,4 @@ DIST git-cola-3.12.0.tar.gz 1004892 BLAKE2B 9396768256341e59500dffc95136e710b2d3
 DIST git-cola-4.0.1.tar.gz 1018819 BLAKE2B 523264bbee38526d12db5ad975d78c72ef6c0dae98566f5a8b5d51b115d222c8aa6daab838ba67fa4c697391c8f58d4b05ddb7920bf65913d217f562aabd352f SHA512 c02150997982c1f8ce65a5f021364fc4c346339f5be4104f82ba6f7688cc05bad3d764e2cf920a22c5f209dc4b857c789907ff58c0d2809bc7ca7c3481b045fd
 DIST git-cola-4.0.2.tar.gz 1022362 BLAKE2B 6f1d6aa6fa6c308f28150018eea8a110c2ceb35ac5e904844568a503e1ebfdb07d9d79b7fa6542aa734c62ae5e5b2e41bb26c99b23a7a84067c102598354cec1 SHA512 01a68caf12bff400b243a57a9b5a5f07c2c1bd1b90d50acc75806ffdd9377409a56dd591311667afab7508f1ff0d88e31744c1990db3f5f6ce9d977f3fa4068e
```

Comment:
Do you think it's possible to drop both 3.12 and 4.0.2?

Reply:
done